supportedBackends/google/pipelines/common/src/main/scala/cromwell/backend/google/pipelines/common/action/ActionUtils.scala

```diff
@@ -10,7 +10,7 @@ object ActionUtils {
   val sshEntryPoint = "ssh-server"
 
   /** Port mappings for the ssh container. */
-  val sshPortMappings = Map("22" -> new Integer(22))
+  val sshPortMappings = Map("22" -> Integer.valueOf(22))
```

Comment:
Similarly here the compiler was complaining that `new Integer(n)` is deprecated and `Integer.valueOf(n)` should be used instead

Reply:
It means you can reuse a singleton `Integer(22)` instead of recreating objects each time. As I understand it, it's a tiny optimization, but not a bad idea.